Source/JavaScriptCore/dfg/DFGClobberize.h

```diff
@@ -1247,6 +1247,11 @@ void clobberize(Graph& graph, Node* node, const ReadFunctor& read, const WriteFu
         def(HeapLocation(CheckTypeInfoFlagsLoc, JSCell_typeInfoFlags, node->child1()), LazyNode(node));
         return;
 
+    case HasStructureWithFlags:
+        read(JSCell_structureID);
+        def(HeapLocation(HasStructureWithFlagsLoc, JSCell_structureID, node->child1()), LazyNode(node));
```

Comment:
This define is not correct because even though JSCell_structureID is not written, the result of this can be changed (because these flags can be changed without structure transitions).